### ISSUE TITLE
Proposal: Add a command to output the list of files credo will analyze

### ISSUE THREAD

**Issue by wasnotrice** [rrrene/credo#514]
### Environment

* Credo version (`mix credo -v`): 0.8.10
* Erlang/Elixir version (`elixir -v`):
```
Erlang/OTP 20 [erts-9.1] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:10] [hipe] [kernel-poll:false]

Elixir 1.6.1 (compiled with OTP 19)
```

* Operating system: MacOS 10.13.3

### What were you trying to do?

I would like to check files in an editor as I make changes (VSCode in this instance, although I suspect the issue might occur in other editors too)

### Expected outcome

When I open a file that is not included in my `.credo.exs` configuration (for example, `mix.exs`), then it won't be checked.

### Actual outcome

Any Elixir file I open is checked.

The [credo plugin for vscode](https://github.com/iampeterbanjo/vscode-elixir-linter) works by sending the current file contents to credo via stdin. Credo has an option to facilitate this, which is great! But there doesn't seem to be a way to ask Credo, "Are you supposed to check this file?" I would like to ask Credo for a list of files that it would check if it were run right now, similar to the output of `find` :

```
$ mix credo files
/full/path/to/lib/supervisor.ex
/full/path/to/lib/rules.ex
/full/path/to/test/supervisor_test.exs
```

Then, if the current file isn't in Credo's list, I wouldn't check it, so my editor would not display Credo warnings. I don't believe this would add any core functionality to Credo. It would just output the list of files that Credo already generates.

Is this a feature you would consider including? If so, I'd be happy to work it up.

**Comment by rrrene** [comment]
`mix credo info` was added in `0.9.0-rc3` in February for this kind of use-case. Based on your proposal, I added a `--verbose` flag, which gives you additional information about Credo's runtime environment.

It is now live on `master`. You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

and using this command:

```
mix credo info --verbose --format=json
```

Please report back if your "issue" is "solved"! 😄👍

**Comment by wasnotrice** [comment]
@rrrene lovely, that's just what I was looking for, thank you! ❤️
